skbio/io/format/_sequence_feature_vocabulary.py

```diff
@@ -255,6 +255,18 @@ def _parse_loc_str(loc_str):
                 fuzzy_e = True
             bounds.append((int(start)-1, int(end)))
             fuzzy.append((fuzzy_s, fuzzy_e))
+        elif p == 'EL':
+            entry, interval = v.split(":")
+            start, end = interval.split('..')
+            fuzzy_s = fuzzy_e = False
+            if start.startswith('<'):
+                start = start[1:]
+                fuzzy_s = True
+            if end.startswith('>'):
+                end = end[1:]
+                fuzzy_e = True
+            bounds.append((int(start)-1, int(end)))
+            fuzzy.append((fuzzy_s, fuzzy_e))
```

Comment:
I see you incorporated you previous PR #1490 here. Pls see my comment in that PR and remove it from this PR.